docs/README.md

```diff
@@ -31,10 +31,10 @@ to [the main OpenTF CLI documentation](https://www.placeholderplaceholderplaceho
   This documentation is for SDK developers, and is not necessary reading for
   those implementing a provider using the official SDK.
 
-* [How OpenTF Uses Unicode](./unicode.md): an overview of the various
-  features of OpenTF that rely on Unicode and how to change those features
+* [How OpenTofu Uses Unicode](./unicode.md): an overview of the various
+  features of OpenTofu that rely on Unicode and how to change those features
   to adopt new versions of Unicode.
 
 ## Contribution Guides
 
-* [Contributing to OpenTF](../.github/CONTRIBUTING.md): a complete guideline for those who want to contribute to this project.
+* [Contributing to OpenTofu](../.github/CONTRIBUTING.md): a complete guideline for those who want to contribute to this project.
```

Comment:
This one was fixed by https://github.com/opentofu/opentofu/pull/445
the link is broken